pkgs/development/python-modules/uarray/default.nix

```diff
@@ -58,6 +65,6 @@ buildPythonPackage rec {
     description = "Universal array library";
     homepage = "https://github.com/Quansight-Labs/uarray";
     license = licenses.bsd0;
-    maintainers = [ ];
+    maintainers = [ pbsds ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ pbsds ];
```

Reply:
i really should have a `meta` check pre-commit hook